tests/cases/compiler/noParameterReassignmentJSIIFE.ts

```diff
@@ -0,0 +1,9 @@
+// @allowJs: true
+// @checkJs: true
+// @noEmit: true
+// @filename: index.js
+self.importScripts = (function (importScripts) {

```

Comment:
I see one big source of my confusion: I forgot that `self: Window & typeof globalThis`, and kept wondering where `importScripts` was declared. This test would be a lot clearer without relying on the DOM.